data/json/monsters/power_leech.json

```diff
@@ -0,0 +1,188 @@
+    "death_drops": {  },
+    "death_function": [ "NORMAL" ],
+    "flags": [ "SEES", "NOHEAD", "ELECTRONIC", "IMMOBILE", "NO_BREATHE", "HARDTOSHOOT" ]
+  },
+  {
+    "id": "mon_leech_pod_cluster",
+    "type": "MONSTER",
+    "name": "leech pod cluster",
+    "description": "The translucent egg pods of an alien plant, firmly attached by luminous rhizomes.  You can barely distinguish a root drone floating withing a cloudy substance.",
+    "default_faction": "nether",
+    "species": [ "LEECH_PLANT" ],
+    "volume": "30000 ml",
+    "weight": "60 kg",
+    "hp": 40,
+    "speed": 100,
+    "material": [ "flesh" ],
+    "symbol": "g",
+    "color": "white",
+    "aggression": 100,
+    "morale": 100,
+    "armor_bash": 15,
+    "luminance": 60,
+    "starting_ammo": { "9mm": 100 },
+    "special_attacks": [
+      {
+        "type": "gun",
+        "cooldown": 15,
+        "gun_type": "emp_frond",
+        "ranges": [ [ 0, 1, "DEFAULT" ] ],
+        "description": "Lightning arcs towards you!"
+      },
+      [ "LEECH_SPAWNER", 35 ]
+    ],
+    "death_drops": {  },
+    "death_function": [ "NORMAL" ],
+    "flags": [ "SEES", "NOHEAD", "IMMOBILE", "NO_BREATHE" ]
+  },
+  {
+    "id": "mon_leech_root_runner",
+    "type": "MONSTER",
+    "name": "root runner",
+    "description": "This clump of woody vegetation hastily clambers around in a lizard-like fashion.  Three translucent scale-leaves stand tall on the backside of the creature, and the thin ridges within them periodically glow through some unknown mean.  Its seemingly a symbiote of the nearby alien ferns, and looks ready to defend them with its life.",
+    "default_faction": "nether",
+    "species": [ "LEECH_PLANT" ],
+    "volume": "4000 ml",
+    "weight": "6 kg",
+    "hp": 40,
+    "speed": 120,
+    "material": [ "flesh" ],
+    "symbol": "m",
+    "color": "blue",
+    "aggression": 100,
+    "morale": 100,
+    "melee_skill": 4,
+    "melee_dice": 1,
+    "melee_dice_sides": 2,
+    "armor_bash": 15,
+    "luminance": 60,
+    "vision_day": 30,
+    "vision_night": 10,
+    "melee_damage": [ { "damage_type": "electric", "amount": 4 } ],
+    "upgrades": { "half_life": 9999, "into": "mon_leech_stalk" },
+    "special_attacks": [
+      {
+        "type": "gun",
+        "cooldown": 15,
+        "gun_type": "emp_frond",
+        "fake_skills": [ [ "gun", 1 ], [ "rifle", 1 ] ],
+        "ranges": [ [ 0, 2, "DEFAULT" ] ],
+        "description": "Lightning arcs towards you!"
+      },
+      [ "EVOLVE_KILL_STRIKE", 3 ]
+    ],
+    "death_drops": {  },
+    "death_function": [ "NORMAL" ],
+    "flags": [ "SEES", "NOHEAD", "ELECTRONIC", "NO_BREATHE", "HARDTOSHOOT" ]
+  },
+  {
+    "id": "mon_leech_root_drone",
+    "type": "MONSTER",
+    "name": "root drone",
+    "description": "A small bulb with a beak like protuberance, skittishly roaming about under three tendril rhizomes.  Dripping and glistening, it resembles a creature newly born rather than a sapling grown from seeds.",
```

Comment:
```suggestion
    "description": "A small bulb with a beak-like protuberance, skittishly roaming about under three tendril rhizomes.  Dripping and glistening, it resembles a creature newly born rather than a sapling grown from seeds.",
```